custom_domains/documentation/aks-logs.md

```diff
@@ -0,0 +1,58 @@
+# Retrieving Log Analytics Data with KQL for AKS Clusters
+
+Kusto Query Language (KQL) is a query language that you can use to retrieve data from Log Analytics workspaces for Azure Kubernetes Service (AKS) clusters. In this guide, we will explain how to write KQL queries to retrieve Log Analytics data and analyze it for your AKS cluster.
+
```

Comment:
Maybe add a warning that it takes a few minutes for the logs to appear in tbhe workspace